mchf-eclipse/drivers/audio/audio_driver.c

```diff
@@ -3710,7 +3710,7 @@ static void AudioDriver_RxProcessor(AudioSample_t * const src, AudioSample_t * c
             //
             if(dmod_mode != DEMOD_SAM && dmod_mode != DEMOD_AM) // || ads.sam_sideband == 0) // for SAM & one sideband, leave out this processor-intense filter
             {
-                if(ts.filter_path < 48)
+                if(ts.filter_path < 48 && dmod_mode != DEMOD_FM)
```

Comment:
Hi Danilo,
yes, you are absolutely right. That was not well done from my side.
I would put it like this:
                if(FilterPathInfo[ts.filter_path].FIR_I_coeff_file == i_rx_new_coeffs && dmod_mode != DEMOD_FM)
Reason is that we need to decimate-by-4 before applying the Hilberts, if we use the new coefficients (all SSB & CW filters below 3k8) and we decimate AFTER the Hilberts, if we use the old coeffs. 
Thanks!
73 de Frank